core-java/src/test/java/com/baeldung/nulls/UsingOptionalUnitTest.java

```diff
@@ -0,0 +1,37 @@
+package com.baeldung.nulls;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Optional;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class UsingOptionalUnitTest {
+
+    private UsingOptional classUnderTest;
+
+    @BeforeEach
+    public void setup() {
+        classUnderTest = new UsingOptional();
+    }
+
+    @Test
+    public void whenArgIsFalse_thenReturnEmptyResponse() {
+        Optional<Object> result = classUnderTest.process(false);
+        assertFalse(result.isPresent());
```

Comment:
leave a blank line between the given and when parts and one between the when and then parts of tests.